src/hooks/__tests__/useFullScreen.spec.tsx

```diff
@@ -0,0 +1,48 @@
+import { act, renderHook } from '@testing-library/react';
+import { useFullScreen } from '../custom-hooks';
+
+describe('useFullScreen hook', () => {
```

Comment:
nit: from how we do test cases for p2p you can just put the name of the hook/component

```suggestion
describe('useFullScreen', () => {
```